packages/application-shell/src/components/custom-view-shell/custom-view-shell.tsx

```diff
@@ -116,6 +118,20 @@ function StrictModeEnablement(props: TStrictModeEnablementProps) {
   }
 }
 
+function CustomViewThemeProvider() {
+  const theme = useFeatureToggle('mcRecolouring') ? 'recolouring' : 'default';
+
+  const customViewThemeOverrides = {
+    ...themesOverrides[theme],
+    // @ts-ignore
```

Comment:
This is temporary until the [recolouring PR](https://github.com/commercetools/merchant-center-application-kit/pull/3414) is merged.